### PR TITLE
BYOC to Multi-Account

### PR DESCRIPTION
**Pull request by Oshratn**
Change the naming of BYOC to multi-account support

Towards [#9645](https://github.com/giantswarm/giantswarm/issues/9645)

### CODE REVIEW

File: src/content/reference/gsctl/update-org-set-credentials.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-date: 2018-09-12
+date: 2020-05-19
 type: page
 weight: 55
 ---
 
 # `update organization set-credentials`
 
-Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We call this by the catchy term [Bring Your Own Cloud](/basics/byoc/) (BYOC). As a prerequisite, the organizaiton that should own the clusters has to be prepared with cloud provider credentials.
+Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We refer to this as [Multi Account functionality](/basics/byoc/), renamed from Bring Your Own Cloud (BYOC). As a prerequisite, the organizaiton that should own the clusters has to be prepared with cloud provider credentials.
```
Comment (puja108) [inline src/content/reference/gsctl/update-org-set-credentials.md:11]: are we not changing the link? `/byoc/`

Comment (puja108) [reply src/content/reference/gsctl/update-org-set-credentials.md:11]:
```suggestion
Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We refer to this as [Multi Account functionality](/basics/multi-account/), renamed from Bring Your Own Cloud (BYOC). As a prerequisite, the organization that should own the clusters has to be prepared with cloud provider credentials.
```

Comment (puja108) [reply src/content/reference/gsctl/update-org-set-credentials.md:11]: btw, did you rename that already? need to merge the changes together

Comment (Oshratn) [reply src/content/reference/gsctl/update-org-set-credentials.md]: I did rename it [PR #506](https://github.com/giantswarm/docs/pull/506)

**Comment by Oshratn** [comment]
@marians this is still blocked. Is it just a matter of an additional review?

**Comment by marians** [comment]
@Oshratn Don't you get a button like this?

image

**Comment by Oshratn** [comment]
I do now 😃

**Pull request merged**